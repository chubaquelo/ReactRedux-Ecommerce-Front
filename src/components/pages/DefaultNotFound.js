import React from 'react';

const DefaultNotFound = () => (
  <h1 className="h-70percent-vh pt-5 text-center text-3xl text-red-500 font-bold">
    Error 404 - Not Found
  </h1>
);

export default DefaultNotFound;
