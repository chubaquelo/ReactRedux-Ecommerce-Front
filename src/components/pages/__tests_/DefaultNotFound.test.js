import React from 'react';
import renderer from 'react-test-renderer';
import DefaultNotFound from '../DefaultNotFound';

describe('renders correctly', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<DefaultNotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
