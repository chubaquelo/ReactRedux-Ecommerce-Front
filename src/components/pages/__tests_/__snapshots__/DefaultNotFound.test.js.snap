// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly renders correctly 1`] = `
<h1
  className="h-60percent-vh pt-20 text-center text-3xl text-red-500 font-bold"
>
  This page is not implemented. Sorry.
</h1>
`;
